libs/librrgraph/src/base/rr_node_types.h

```diff
@@ -128,3 +128,45 @@ struct t_rr_rc_data {
 // This is the data type of fast lookups of an rr-node given an (rr_type, layer, x, y, and the side)
 //[0..num_rr_types-1][0..num_layer-1][0..grid_width-1][0..grid_height-1][0..NUM_2D_SIDES-1][0..max_ptc-1]
 typedef vtr::array<e_rr_type, vtr::NdMatrix<std::vector<RRNodeId>, 4>, (size_t)e_rr_type::NUM_RR_TYPES> t_rr_node_indices;
+
+/**
```

Comment:
This is better than vpr_types.h.  But should it be in a route_common.h or router.h file instead of here?